modules/platform/nuxeo-drive-server/nuxeo-drive-core/src/test/java/org/nuxeo/drive/fixtures/AuditChangeFinderClusteringEnabledTestSuite.java

```diff
@@ -34,14 +34,12 @@
 import org.nuxeo.drive.service.FileSystemItemChange;
 import org.nuxeo.ecm.core.api.DocumentModel;
 import org.nuxeo.ecm.core.api.impl.blob.StringBlob;
-import org.nuxeo.runtime.test.runner.Deploy;
 
 /**
  * Tests the {@link FileSystemChangeFinder} on a repository with clustering enabled.
  *
  * @since 8.2
  */
-@Deploy("org.nuxeo.drive.core:OSGI-INF/test-nuxeodrive-clustering-enabled-repo-contrib.xml")
```

Comment:
This shouldn't be removed, otherwise clustering isn't enabled for this test suite.

Reply:
back in place, now the problem is the following:
the query transferred to MongoDB is in the form 
```
{"repositoryId": {"$in": ["test"]}, "logDate": {"$lt": {"$date": 1598265956818}}}
```
which doesn't work consistently

If I send the same date in the mongo shell as an ISODate
```
{"repositoryId": {"$in": ["test"]}, "logDate": {"$lt": ISODate("2020-08-24T12:45:56.818556+02:00")}}
```
it will work.

**Now the really inconsistent part:**
The date sent is given by the formula: now - 2 * clusteringDelay as milliseconds 🔴 
If I give the date without substracting 2 * clusteringDelay as milliseconds it will work 🟢
If I give the date without substracting 2 * clusteringDelay as **NANOSeconds** it will work 🟢 

**Now more inconsistency:**
I have faced the same problem with upper and lower bound:
The query sent by the java driver was of the form:
{"lowerbound": {"$gt": {"$numberLong": "0"}}} 🟢 
The same query copied in the mongo shell would fail 🔴 
in the mongoShell I had to write: {"lowerbound": {"$gt": 0}} 🟢